src/api-reference/user-provisioning/spend/spend-role-code-definition.md

```diff
@@ -152,12 +151,12 @@ The role code value that can be used for provisioning with the `Spend Role Exten
 |`REPORTING_CAS_ANALYST`|CAS Analyst|**NOTE: Do not use. This role is retired.**|Y|N|
-|`REPORTING_BUSINESS_AUTHOR`|Cognos Business Author|The user assigned this role can use Concur Insight to view data for reports submitted at all hierarchical levels within their assigned groups (as well as any data not group-related). The user is assigned the Business license type, which restricts the features to which they have access. They may also run existing reports, create new reports or modify existing basic reports using the basic tool, Query Studio. <br>**NOTE**: A user cannot be assigned this role and the Cognos Professional Author or Cognos Consumer roles, only one of these three may be assigned.|Y|N|
-|`REPORTING_CONSUMER`|Cognos Consumer|The user assigned this role can use Concur Insight to view data for reports submitted at all hierarchical levels within their assigned groups (as well as any data not group-related). The user is assigned the Consumer license type, which further restricts the features to which they have access. They may also run existing reports, with read-only access. <br>**NOTE**: A user cannot be assigned this role and the Cognos Professional Author or Cognos Business Author roles, only one of these three may be assigned.|Y|N|
-|`REPORTING_PRO_AUTHOR`|Cognos Professional Author|The user assigned this role can use Concur Insight to view data for reports submitted at all hierarchical levels (no group assignment required). The user is assigned the Professional license type, which allows access to all features. They may also run existing reports, create new reports or modify existing basic reports using the basic tool, Query Studio, and create new reports or modify existing basic reports using the advanced tool, Report Studio. They can also schedule automatic report runs.|Y|N|
-|`REPORTING_CONFIG_ADMIN`|Consolidation Configuration Administrator     |This role is used by companies using the Reporting Database that wish to consolidate the data from the current version and previous versions.<br>The user assigned this role can map expense types and custom fields to a single consolidated definition, and adjust other consolidation settings. <br>**NOTE:** This role does not include access to Concur Insight.|N|N|
+|`REPORTING_BUSINESS_AUTHOR`|Cognos Business Author|The user assigned this role can use Analysis/Intelligence to view data for reports submitted at all hierarchical levels within their assigned groups (as well as any data not group-related). The user is assigned the Business license type, which restricts the features to which they have access. They may also run existing reports, create new reports or modify existing basic reports using the basic tool, Query Studio. <br>**NOTE**: A user cannot be assigned this role and the Cognos Professional Author or Cognos Consumer roles, only one of these three may be assigned.|Y|N|
```

Comment:
Do we need to define terms like "Analysis/Intelligence" in terms of what they mean within concur? or are people kinda expected to know what these things around prior to coming here and already somewhat know what they are looking for. I for one don't know anything about these two things...

Reply:
For same area, it seems sometimes you bold the roles that cannot be also be assigned. Should `Cognos Professional Author` and `Cognos Consumer roles` be bolded?